CSharpSourceCode/AttributeDataSystem/PartyAttribute.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System.Collections.Generic;
+using TaleWorlds.CampaignSystem;
+using TaleWorlds.SaveSystem;
+
+namespace TOW_Core.AttributeDataSystem
+{
+    /// <summary>
+    /// Contains TOW troop data of a complete mobile Party. For single entity data use a StaticAttribute
+    /// </summary>
+    public class PartyAttribute
+    {
+        [SaveableField(1)]
+        public Hero Leader;
+        [SaveableField(2)]
+        public string id;           //partybase.
```

Comment:
What is this `//partybase.` comment for? If it's meaningful, can you add more information? It's not clear to me what it's saying. If it's a "foreign key" to a Party, maybe rename the variable to something like `partyId` and remove the comment.

Reply:
done